docs/badges/umd.svg

```diff
@@ -14,7 +14,7 @@
   <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
     <text x="16.5" y="15" fill="#010101" fill-opacity=".3">umd</text>
     <text x="16.5" y="14">umd</text>
-    <text x="59" y="15" fill="#010101" fill-opacity=".3">21.33 kB</text>
-    <text x="59" y="14">21.33 kB</text>
+    <text x="59" y="15" fill="#010101" fill-opacity=".3">21.41 kB</text>
+    <text x="59" y="14">21.41 kB</text>
```

Comment:
I have no idea why this actually went up... As far as I know, the umd build shouldn't have the test logic at all :thinking: